common/perf-test-core/src/main/java/com/azure/perf/test/core/PerfStressProgram.java

```diff
@@ -251,13 +269,13 @@ public static void runTests(PerfTestBase<?>[] tests, boolean sync, int parallel,
                 });
 
                 Flux.range(0, parallel)
-                    .parallel()
-                    .runOn(Schedulers.boundedElastic())
-                    .flatMap(i -> tests[i].runAllAsync(endNanoTime))
+                    .parallel(parallel)
+                    .runOn(Schedulers.parallel())
+                    .flatMap(i -> tests[i].runAllAsync(endNanoTime), false, Math.min(parallel, 1000 / parallel), 1)
```

Comment:
I think the `Math.min(parallel, 1000 / parallel)` might be incorrect.

Reply:
This will be removed in #32020